Install.md

```diff
@@ -1,321 +1,224 @@
+# Install and utilization notes for Unix users
```

Comment:
Maybe we should provide the big lines of what is being installed ? A server, a client, both ? In what purpose ? What will this demonstrate/provide ?
Also clearly separate what concerns the server side from the client side.

Reply:
You're right, I added a short description and some links to answer these questions.